test/foundry/core/Allo.t.sol

```diff
@@ -500,9 +506,93 @@ contract AlloTest is Test, AlloSetup, RegistrySetupFull, Native, Errors, GasHelp
         bytes[] memory datas = new bytes[](1);
         datas[0] = bytes("data1");
 
+        uint256[] memory values = new uint256[](2);
+        values[0] = 100;
+        values[1] = 20;
```

Comment:
Can you add this values as constant in the setup? And maybe you can `vm.deal(makeAddr("anon"), 100);` too